.github/workflows/ci.yml

```diff
@@ -14,5 +14,5 @@ on:
 
 jobs:
   call-ci:
-    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-submodule.yml@master
+    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-submodule.yml@KArkadiusz/aarch64_ci_test
```

Comment:
We rather do such temporary changes in the separate commit, marking that they won't be added to the project and it's only to check the result on CI before merge.